packages/base/src/keybindings.json

```diff
@@ -2,22 +2,42 @@
   {
     "command": "jupytercad:undo",
     "keys": ["Accel Z"],
-    "selector": "body"
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:undo",
+    "keys": ["Accel Z"],
+    "selector": ".jpcad-sidepanel-widget"
   },
   {
     "command": "jupytercad:redo",
     "keys": ["Accel Y"],
-    "selector": "body"
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:redo",
+    "keys": ["Accel Y"],
+    "selector": ".jpcad-sidepanel-widget"
   },
   {
     "command": "jupytercad:removeObject",
     "keys": ["Delete"],
-    "selector": "body"
+    "selector": ".jcad-Mainview"
+  },
+  {
+    "command": "jupytercad:removeObject",
+    "keys": ["Delete"],
+    "selector": ".jpcad-sidepanel-widget"
+  },
+  {
+    "command": "jupytercad:wireframe",
+    "keys": ["Accel Space"],
+    "selector": ".jcad-Mainview"
   },
```

Comment:
I tried registering it as  -
```
{
    "command": "jupytercad:removeObject",
    "keys": ["Delete"],
    "selector": ".jcad-Mainview .jpcad-sidepanel-widget"
}
```

But strangely it was not working for any of this in this case!